packages/next/lib/get-network-hosts.ts

```diff
@@ -0,0 +1,16 @@
+import os from 'os'
+
+export function getNetworkHost(): string | null {
+  const interfaces = os.networkInterfaces()
+  let host: string | null = null
+
+  Object.keys(interfaces).forEach((key) =>
+    interfaces[key]
+      ?.filter(({ family, address }) => {
+        return family === 'IPv4' && !address.includes('127.0.0.1')
```

Comment:
```suggestion
        return family === 'IPv4' && !address.includes('127.0.0.1')
```

Most platforms would use `127.0.0.1`, but theoretically, the entire `127.0.0.0/8` should be excluded. There are environments that have more loopback IPs bound to multiple interfaces.